doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -95,6 +95,9 @@ Security
 
 |no_changes_yet_note|
 
+Deprecated:
+* Deprecated legacy Mbed TLS crypto toolbox APIs that are enabled when :kconfig:option:`CONFIG_NORDIC_SECURITY_BACKEND` Kconfig is set. Please use :kconfig:option:`CONFIG_NRF_SECURITY` and PSA crypto APIs instead.
+
```

Comment:
```suggestion

* Deprecated:

  * Legacy Mbed TLS crypto toolbox APIs that are enabled when the :kconfig:option:`CONFIG_NORDIC_SECURITY_BACKEND` Kconfig option is set.
    Use the nRF Security (enabled with the :kconfig:option:`CONFIG_NRF_SECURITY` Kconfig option) and PSA crypto APIs instead.

```

Reply:
This changelog entry should be removed from your PR, and added as a change request to PR: https://github.com/nrfconnect/sdk-nrf/pull/18192

Reply:
I have added this change here: https://github.com/nrfconnect/sdk-nrf/compare/41f93f0b887ebeb166cbf4c7d24cd398642d841c..195d67075aa037c6b435111d26a223b35ded9ce1 . Please remove this entry from this PR.